### ISSUE TITLE
[Bug] Cubari: Gist chapter download issue - kotlinx.serialization.json.JsonLiteral is not a JsonArray

### ISSUE THREAD

**Issue by connect192168** [tachiyomiorg/tachiyomi-extensions#7429]
## Device information
- Tachiyomi version: Stable 0.11.1
- Android version: 11

## Source information
- Name: Cubari
- Extension version: 1.2.6

## Steps to reproduce
1. Open [cubari:gist/CheatCode](https://cubari.moe/read/gist/CheatCode/) in Tachiyomi
2. Attempt to download any of the three chapters.

### Expected behavior
The chapter should download successfully.

### Actual behavior
The dowload fails with the error:
`Element class kotlinx.serialization.json.JsonLiteral is not a JsonArray`

### An educated guess about the cause by someone who has written less than 50 lines of java in his life.
I'm going to make an educated guess that this was caused by  [PR 7398](https://github.com/tachiyomiorg/tachiyomi-extensions/pull/7398) which was merged approx. 19h ago and switched cubari from gson to kotlinx json serialization.
I might be wrong, but if I'm right then someone with more java experience will have somewhere to start digging when attempting to fix the problem.

**Comment by alessandrojean** [comment]
Thanks for providing an example, I will take a look.

**Comment by CarlosEsco** [comment]
@connect192168 don't be a douche

**Comment by connect192168** [comment]
> @connect192168 don't be a douche

I'm not trying to be.
Re-reading my bug report, that trailing sentence does read like I'm being a bit miserable.
That was not my intention and I apologise for it coming of that way (and not using grammaer where I should have).

I just figured I'd make an educated guess in case it was helpful to whoever tries to fix the problem.
I'm more than well aware that the people putting time and effort into this know waaaaay more than I do about java, so I tried to put a bit of leg work in to hopefully save someone time when trying to fix the issue.
